integration/testnet/network.go

```diff
@@ -609,10 +615,13 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig, chainID flow.Ch
 	})
 
 	// create a temporary directory to store all bootstrapping files
-	bootstrapDir, err := integrationBootstrapDir()
-	require.Nil(t, err)
+	bootstrapDir := tempDir(t)
 
 	t.Logf("BootstrapDir: %s \n", bootstrapDir)
+	t.Cleanup(func() {
```

Comment:
Do we need this since we add clean up in  https://github.com/onflow/flow-go/pull/3846/files#diff-7540be57ca1fb2714cfce630ad0eba7bfc97362d8e30b79ac2551dc68792e75bR585 ?

Reply:
👍 removed